packages/api/src/StyleOptions.ts

```diff
@@ -259,6 +259,16 @@ type StyleOptions = {
   microphoneButtonColorOnDictate?: string;
   sendBoxBackground?: string;
 
+  /**
+   * The comma-delimited file types that the upload button should accept.
+   * @example 'image/*,.pdf'
+   */
+  uploadFileTypes?: string;
```

Comment:
Maybe call this `uploadAccept` to match its HTML attributes `accept`.

Having a plural word but not accepting an array feels weird.